pkg/hints/hint_utils/bigint_utils.go

```diff
@@ -0,0 +1,95 @@
+package hint_utils
+
+import (
+	"math/big"
+
+	. "github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
+	. "github.com/lambdaclass/cairo-vm.go/pkg/vm"
+	. "github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+	"github.com/pkg/errors"
+)
+
+// In cairo, various structs are used to represent big integers, all of them have numbered fields of Felt type (d0, d1,...) and share the same behaviours
+// This file contains an implementation of each behaviour at the limbs level, and the wrappers for each specific type
+
+// Generic methods for all types
+func limbsFromVarName(nLimbs int, name string, ids IdsManager, vm *VirtualMachine) ([]Felt, error) {
+	baseAddr, err := ids.GetAddr(name, vm)
+	if err != nil {
+		return nil, err
+	}
+	return limbsFromBaseAddress(nLimbs, name, baseAddr, vm)
+}
+
+func limbsFromBaseAddress(nLimbs int, name string, addr Relocatable, vm *VirtualMachine) ([]Felt, error) {
+	limbs := make([]Felt, 0)
+	for i := 0; i < nLimbs; i++ {
+		felt, err := vm.Segments.Memory.GetFelt(addr.AddUint(uint(i)))
+		if err == nil {
+			limbs = append(limbs, felt)
+		} else {
+			return nil, errors.Errorf("Identifier %s has no member d%d", name, i)
+		}
+	}
+	return limbs, nil
+}
+
+func limbsPack86(limbs []Felt) big.Int {
+	sum := big.NewInt(0)
+	for i := 0; i < 3; i++ {
```

Comment:
I believe this should be 

```
for i := 0; i < len(limbs); i++ {
```

Reply:
https://github.com/lambdaclass/cairo-vm/blob/8d711e136d4ad590249975f0b750502c44b8177b/vm/src/hint_processor/builtin_hint_processor/secp/bigint_utils.rs#L88

Reply:
Interesting 🤔